ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.ts

```diff
@@ -21,7 +24,10 @@ export class IndividualJourney extends JourneyBase {
   private isDone: boolean;
   private isSubmitted: boolean;
 
-  constructor(private individualStepsOrderFactory: IndividualStepsOrderFactory) {
+  private stepsWithAnswers: Array<MutableIndividualSuitabilityModelWithStep> = [];
```

Comment:
Looks like you can remove this? (and the class?)

Reply:
removed the code, class